pkg/kn/commands/source/apiserver/apiserver.go

```diff
@@ -26,8 +26,8 @@ import (
 // NewAPIServerCommand for managing ApiServer source
 func NewAPIServerCommand(p *commands.KnParams) *cobra.Command {
 	apiServerSourceCmd := &cobra.Command{
-		Use:   "apiserver",
-		Short: "Kubernetes API Server Event Source command group",
+		Use:   "apiserver COMMAND",
```

Comment:
Shouldn't this be `api-server COMMAND`?

Reply:
We settled on `apiserver` for the command (like in `kn source apiserver list`), so changing it now would be backwards incompatible. We could do this if its improve the UX considerably but should be aware that it involves more work than just renaming it here. 

What we can do though is to add an alias `api-server` for `apiserver` alongside the suggestions in #812 .

Reply:
:+1 for alias